libs/dal/src/repositories/message/message.repository.ts

```diff
@@ -91,7 +91,7 @@ export class MessageRepository extends BaseRepository<MessageDBModel, MessageEnt
     subscriberId: string,
     channel: ChannelTypeEnum,
     query: { feedId?: string[]; seen?: boolean; read?: boolean } = {},
-    options: { limit: number } = { limit: 1000 } // todo NV-2161 update to 100 in version 0.16
+    options: { limit: number; skip?: number } = { limit: 100, skip: 0 }
```

Comment:
We need to start to centralise these in shared constants so we avoid to miss them when updating. One for the tech debt.